### DISCUSSION TITLE
GitHub API bug

### DISCUSSION THREAD

**Discussion by luisotaviocap98** [community/community#4968] [Projects and Issues]
Hello everybody. I am new here. I'm using the github API for a university course and I noticed an error. I couldn't find any forum or repository to post as an Issue. But on return from API, field "watchers_count" is returning wrong value, it returns the same value as "stargazers_count". For example, when querying [https://api.github.com/repos/github/feedback](https://api.github.com/repos/github/feedback), the "watchers_count" field should return 26.

**Comment by jiacai2050** [comment]
It's not a bug.
- https://github.community/t/bug-watchers-always-same-as-stargazers-count-in-api-user-starred/14224/2?u=jiacai2050
